test/ui-test/src/screens/StatusChatScreen.py

```diff
@@ -53,6 +63,30 @@ def verify_last_message_sent_is_not(self, message: str):
             test.passes("Success: No message was found")
             return
         verify_text_does_not_contain(str(last_message_obj.text), str(message))
+    
+    # This method expect to have just one mention / link in the last chat message 
+    def verify_last_message_sent_contains_mention(self, displayName: str, message: str):
+        [loaded, last_message_obj] = is_loaded_visible_and_enabled(ChatComponents.LAST_MESSAGE_TEXT.value)
+        
+        if loaded:
+            # Verifying mention
+            verify_text_contains(str(last_message_obj.text), displayName)
+            
+            # Verifying message
+            verify_text_contains(str(last_message_obj.text), message)
+            
+            # Get link value from chat text:
+            try:
+                href_info = re.search(_LINK_HREF_REGEX, str(last_message_obj.text)).group(1)
+            except AttributeError:
+                # <a href=, "> not found in the original string
+                verify_failure("Mention link not found in last chat message.")
+            
+            click_link(ChatComponents.LAST_MESSAGE_TEXT.value, href_info)
+            verify(is_found(ChatComponents.MENTION_PROFILE_VIEW.value), "Checking user mentioned profile popup is open.")            
```

Comment:
It should be more generic as it is shown in both cases, pass or fail. Here the squish docu:

`If the optional message parameter (of type string—e.g., a comment, bug number, or test plan reference) is present, it will be added to the test result whether the test passed or failed.`

https://doc.froglogic.com/squish/latest/rgs-squish.html#test.verify-function